netcore/System.Private.CoreLib/src/System.Threading/Thread.cs

```diff
@@ -59,6 +57,7 @@ partial class Thread
 		private IntPtr suspended_event;
 		private int self_suspended;
 		private IntPtr thread_state;
+
 		private Thread self;
 		private object pending_exception;
 		private object start_obj;
```

Comment:
I realize this is not part of the PR, but`pending_exception` and `start_obj` are swapped between netcore managed and native.

Reply:
Good catch. Maybe should really mechanize this soon.